demo-client/src/test/java/com/illia/service/FileHandlingServiceTest.java

```diff
@@ -59,24 +57,19 @@ public void testSaveFile() throws IOException {
                 .thenReturn(savedFilesDirectoryPrefix);
         var fileName = "fileName";
         var content = "Content".getBytes();
+        var savedFilesDirectoryPath = Path.of(savedFilesDirectoryPrefix);
 
-        var response = fileHandlingService.saveFile(fileName, content);
-        assertTrue(response);
-        var savedFilePath = Path.of(config.getDownloadedFilesDirectoryPrefix(), fileName);
-        assertTrue(Files.exists(savedFilePath));
-        assertArrayEquals(content, Files.readAllBytes(savedFilePath));
-    }
-
-    @BeforeEach
-    public void clearAndCreateDirectory() throws IOException {
-        savedFilesDirectoryPath = Path.of(savedFilesDirectoryPrefix);
-        FileUtils.deleteDirectory(savedFilesDirectoryPath.toFile());
-        Files.createDirectory(savedFilesDirectoryPath);
-    }
+        try {
+            Files.createDirectory(savedFilesDirectoryPath);
```

Comment:
We decided to mock fileUtils with write and delete methods and check their invocation instead of creating and deleting files in test